drivers/node/node.go

```diff
@@ -66,8 +74,8 @@ type Driver interface {
 	// ShutdownNode shuts down the given node
 	ShutdownNode(node Node, options ShutdownNodeOpts) error
 
-	// CheckIfPathExists checks whether the given path is present in the node
-	CheckIfPathExists(path string, node Node, options ConnectionOpts) (bool, error)
+	// FindFiles finds and returns the files for the given path regex and the node
+	FindFiles(path string, node Node, options FindOpts) (string, error)
```

Comment:
Add a default implementation for aws node driver. Without that, the aws tests will start failing with FindFiles() method not supported error.

Reply:
oops missed it.. adding it.